app/views/admin/reservations/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>All Reservations</h1>
+
+Reservations By Status<br>
+
```

Comment:
Nice! We will want to do something similar with user and host for their personal reservations.